log.capnp

```diff
@@ -1996,6 +1996,16 @@ struct LiveTorqueParametersData {
   useParams @12 :Bool;
 }
 
+struct MagnetometerCalibrationData {
```

Comment:
I think we want this in two parts: a `SensorEventData` for the calibrated measurements and perhaps another struct for the calibration data itself for debugging

Reply:
I think maybe not. The final "calibrated measurements" are not the only information needed to get bearing. There is an additional step to adjust for offsets in angle-domain. So the final post-all-calculations value would be the magnetometer bearing. But that's not a sensor data - so maybe it is better as is.